src/Compilers/CSharp/Test/Emit/PDB/TypeDefinitionDocumentTests.cs

```diff
@@ -0,0 +1,314 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Debugging;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.PDB
+{
+    public class TypeDefinitionDocumentTests : CSharpTestBase
```

Comment:
>TypeDefinitionDocumentTests [](http://example.com/codeflow?start=17&length=27)

Do we need to test nested types, enums, and delegates?

Reply:
Thank you for this, line directives were completely b0rked :)